docs/source/Tutorial.md

```diff
@@ -753,24 +721,6 @@ our `orc` Monster, let's create some `Weapon`s: a `Sword` and an `Axe`.
   axe = MyGame.Sample.Weapon.WeaponEnd(builder)
 ~~~
 </div>
-<div class="language-javascript">
-~~~{.js}
```

Comment:
why are these deleted? these are examples of how to use FlatBuffers from JS, which is still required. If you're a JS user and don't care about TS, beyond the fact that you require the extra step to transpile, there should still be JS-centric docs on how to use FlatBuffers.